SECEdgar/crawler.py

```diff
@@ -73,8 +74,19 @@ def _sanitize_date(date):
             if date < 10**7 or date > 10**8:
                 raise TypeError('Date must be of the form YYYYMMDD')
 
+    @staticmethod
+    def _check_cik(cik):
```

Comment:
Why `SecCralwer` should be concerned about checking `CIK`? We should take these checks in a different layer unless this class will become bloated.

We can have a `cik.py` class whose responsibility will be to fetch, check and do all sort of things with CIK. We can pass this object as a dependency on the `SecCrawler`. This will decouple `SecCrawler` from too much responsibility.

Reply:
I am currently working on a CIK file and agree. We need to decouple some things from the crawler.